contracts/DCAStrategies/DCAStrategies/DCAStrategies.sol

```diff
@@ -7,5 +7,5 @@ import './DCAStrategiesPositionsHandler.sol';
 import '../../interfaces/IDCAStrategies.sol';
 
 contract DCAStrategies is DCAStrategiesManagementHandler, DCAStrategiesPermissionsHandler, DCAStrategiesPositionsHandler, IDCAStrategies {
-  constructor() ERC721('Mean Finance - DCA Strategy Position', 'MF-DCA-STRAT-P') {}
+  constructor() ERC721('Mean Finance - DCA Strategy Position', 'MF-DCA-STRAT-P') EIP712('Mean Finance - DCA Strategy Position', '1') {}
```

Comment:
Maybe we chould pass name and symbol to `DCAStrategiesPermissionsHandler` and configure `EIP712` here? Or do you prefer to have it here?

Reply:
For me, It looks more clear in this way, because you see all constructor data in the same line

Reply:
Sounds good, let's leave it here then 👍